sdk/search/Azure.Search.Documents/README.md

```diff
@@ -1,46 +1,42 @@
-# Azure Cognitive Search client library for .NET
+# Azure AI Search client library for .NET
 
-[Azure Cognitive Search](https://docs.microsoft.com/azure/search/) is a
-search-as-a-service cloud solution that gives developers APIs and tools
-for adding a rich search experience over private, heterogeneous content
-in web, mobile, and enterprise applications.
+[Azure AI Search](https://docs.microsoft.com/azure/search/) (formerly known as "Azure Cognitive Search") is an AI-powered information retrieval platform that helps developers build rich search experiences and generative AI apps that combine large language models with enterprise data.
 
-The Azure Cognitive Search service is well suited for the following
+The Azure AI Search service is well suited for the following
  application scenarios:
 
 * Consolidate varied content types into a single searchable index.
   To populate an index, you can push JSON documents that contain your content,
   or if your data is already in Azure, create an indexer to pull in data
   automatically.
 * Attach skillsets to an indexer to create searchable content from images
-  and large text documents. A skillset leverages AI from Cognitive Services
+  and unstructured documents. A skillset leverages APIs from Azure AI Services
   for built-in OCR, entity recognition, key phrase extraction, language
   detection, text translation, and sentiment analysis. You can also add
   custom skills to integrate external processing of your content during
   data ingestion.
 * In a search client application, implement query logic and user experiences
-  similar to commercial web search engines.
+  similar to commercial web search engines and copilot apps.
```

Comment:
Should "Copilot" be updated here, or is it not being used as a brand in this instance? I could see it either way.

Reply:
@heaths yikes, I didn't realize it was a brand (I was using the term generically).  I'll change it to "chat-style apps".